data/themes/darktable.css

```diff
@@ -1258,6 +1257,106 @@ filechooser row:hover,
   border: 0px;
 }
 
+/*--------------------------------
+  - module layouts manager dialog -
+  --------------------------------*/
+
+#modulegroups_editor
+{
+  color: @fg_color;
+  background-color: @plugin_bg_color;
+  padding-left: 0.5em;
+}
```

Comment:
I suggest to add an empty line after between CSS parts. It's help reading CSS by aerate it I think.